pubspec.lock

```diff
@@ -378,7 +378,7 @@ packages:
       name: xml
       url: "https://pub.dartlang.org"
     source: hosted
-    version: "3.7.0"
+    version: "4.2.0"
```

Comment:
Why not using the newest version 4.3.0

Reply:
Just what `pub` resolved when I installed. Running `pub upgrade` bumps version numbers to a looot of packages, and is probably beyond the scope of this PR. 

Have manually edited pubspec.lock to use xml 4.3.0, then ran `pub get`, which also brought in an update to `meta`.